storages/backends/ftp.py

```diff
@@ -84,11 +87,13 @@ def _start_connection(self):
 
         # Real reconnect
         if self._connection is None:
-            ftp = ftplib.FTP()
+            ftp = ftplib.FTP_TLS() if self._secure else ftplib.FTP()
```

Comment:
And then branch the secure pieces based on the config value.

Reply:
Done. Check: https://github.com/jschneier/django-storages/pull/1320/commits/099196a250e943b9ec572b0d52373d4259314672